### DISCUSSION TITLE
Zigbee2mqtt update

### DISCUSSION THREAD

**Discussion by EMP83** [Koenkk/zigbee2mqtt#10698] [General]
Hello,
I can't find instructions to update to the latest version of zigbee2mqtt.
I installed it in hoobs but I don't know how to  update it. I can't find any documentation.

Thanks.

**Comment by lux73** [comment]
https://www.zigbee2mqtt.io/guide/installation/01_linux.html#for-later-update-zigbee2mqtt-to-the-latest-version

**Reply by EMP83** [reply]
Thank you 🤗👍
